cache/redis/cache.go

```diff
@@ -24,8 +29,9 @@ func (c *cache) Get(key string) (resp []byte, ok bool) {
 }
 
 // Set saves a response to the cache as key.
-func (c *cache) Set(key string, resp []byte) {
-	c.conn().Do("SET", cacheKey(key), resp)
+func (c *cache) Set(key string, value []byte) {
+	ttl, _ := strconv.Atoi(env.GetConfig().TTL)
```

Comment:
:red_circle: I don't think this will work, as `strconv.Atoi` [doesn't evaluate expressions](https://play.golang.org/p/DNrLsG9r_Ek), it only parses numbers.

A possible solution is to set the default as the correct number (the one retrieved from the multiplication) and comment it in the struct itself:
```go
// ...
TTL string `default:"432000"` // five days in seconds
// ...
```

Reply:
Nice catch. Completely overlooked that.